app/assets/src/components/views/SampleViewV2/SampleViewV2.jsx

```diff
@@ -399,32 +399,21 @@ export default class SampleViewV2 extends React.Component {
     );
   };
 
-  computeRowContigStats = ({ row, minContigReads }) => {
+  setRowContigStats = ({ row, minContigReads }) => {
```

Comment:
You might not need this function.
You probably just need to adjust the key names in the ReportTable: https://github.com/chanzuckerberg/idseq-web/blob/master/app/assets/src/components/views/SampleViewV2/ReportTable.jsx#L156

Reply:
Thanks for pointing that out. Done.